src/odemis/driver/test/npmc_test.py

```diff
@@ -52,6 +54,11 @@
            },
 }
 
+COMP_ARGS = {
+    "atol": 1e-2,
```

Comment:
These are very very large rounding. If the values are in "m", that would mean +- 1 cm! (But maybe, as you didn't put any conv_factor, it's actually in mm). The moves in the test cases seem to be around "0.1", so that's still 10% error, which sounds really large. Could you at least lower `atol` to 1e-3?